pkg/virt-handler/device-manager/deviceplugin/v1beta1/constants.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright The Kubernetes Authors.
```

Comment:
Keep as is since this was authored by the Kubernetes authors?

Reply:
so as for now we dropped the year for 2 Kubernetes files:
    "pkg/virt-api/webhooks/validating-webhook/admitters/validate-k8s-utils.go" it was 2014
    "pkg/virt-handler/device-manager/deviceplugin/v1beta1/constants.go" it was 2018
    
it looks the same for all years, so that's why I removed the year from it- so it will be up-to-date.

and regarding https://spdx.org/licenses/Apache-2.0.html I do see the year there but it's very old doc and also for kubevirt we decided to remove the year to keep alignment and the need to update it every year..

Reply:
Let's go ahead with this for now, but we should look into the SPDX identifiers in the future.

Reply:
@0xFelix Thanks and fixed the Kubernetes files to be as they was. will appreciate your lgtm (:

and @enp0s3 WDYT on using SPDX-License-Identifier?

Reply:
@avlitman @0xFelix Definitely worth exploring this option over the current one.